winpr/libwinpr/synch/wait.c

```diff
@@ -285,9 +285,9 @@ DWORD WaitForSingleObject(HANDLE hHandle, DWORD dwMilliseconds)
 
 DWORD WaitForSingleObjectEx(HANDLE hHandle, DWORD dwMilliseconds, BOOL bAlertable)
 {
-	WLog_ERR(TAG, "Function not implemented.");
-	assert(0);
-	return WAIT_OBJECT_0;
+	WLog_ERR(TAG, "%s: Not implemented.");
```

Comment:
Did you forget to use __FUNCTION__ macro here and below?
WLog_ERR(TAG, "%s: Not implemented.", __FUNCTION__);